README.md

```diff
@@ -74,6 +74,8 @@ Use these environment variables to change the default behaviour of the container
 |`-e DECONZ_VNC_MODE=1`|Set this option to enable VNC access to the container to view the deCONZ ZigBee mesh|
 |`-e DECONZ_VNC_PORT=5900`|Default port for VNC mode is 5900; this option can be used to change this port|
 |`-e DECONZ_VNC_PASSWORD=changeme`|Default password for VNC mode is 'changeme'; this option can (should) be used to change the default password|
+|`-e DECONZ_VNC_PASSWORD_FILE=/var/run/secret`|Per default this is disabled and DECONZ_VNC_PASSWORD is used. More details on secrets can be found in the [corresponding section from the official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets) |
```

Comment:
Is this the right link, or should we point users to: https://docs.docker.com/engine/swarm/secrets/.

It would appear also that the correct place that the secret would be located is `/run/secrets/<secret_name>` per that link.

"Details on creating secrets for use with Docker containers can be found in the [corresponding section from the official documentation]..."

Reply:
I'll update the URL to https://docs.docker.com/engine/swarm/secrets/ and the example path to /var/secrets/my_secret